src/showingpreviously/cinemas/isle_of_bute_discovery_centre_cinema.py

```diff
@@ -0,0 +1,95 @@
+import re, lxml, datetime
```

Comment:
Please have one import per line. And `lxml` is not used in this file.

Reply:
Changed to one import per line. And `lxml` is used for parsing the RSS feed using BeautifulSoup:
```
feed_soup = BeautifulSoup(feed_contents, features='xml')
```